packages/main/test/specs/DateRangePicker.spec.js

```diff
@@ -215,4 +215,15 @@ describe("DateRangePicker general interaction", () => {
 		assert.strictEqual(monthButton.innerHTML, monthName, "The month is not changed after selecting the first date in the future");
 	});
 
+	it("Change event is fired with single value", async () => {
+		await browser.url(`test/pages/DateRangePicker.html`);
+		const daterangepicker = await browser.$("#daterange-picker1");
+
+		await daterangepicker.click();
+		await daterangepicker.keys("7 Sep 2022")
+		await daterangepicker.keys("Enter");
+
+		assert.equal(await daterangepicker.getProperty("value"), "7 Sep 2022", "value is updated");
```

Comment:
can you also assert the startDateValue endDateValue as they are public methods and directly related to this use-case.

It's a question what should endDateValue return and if it works at all